core/src/subgraph/registrar.rs

```diff
@@ -455,6 +455,14 @@ where
         Ok(())
     }
 
+    async fn remove_deployment(&self, id: &DeploymentHash) -> Result<(), SubgraphRegistrarError> {
+        self.store.drop_subgraph(id)?;
+
+        debug!(self.logger, "Removing deployment(s)"; "hash" => id.to_string());
+
+        Ok(())
+    }
```

Comment:
Was the debug logs supposed to be before the  `self.store.drop_subgraph(id)?;` ?